README.adoc

```diff
@@ -122,16 +93,18 @@ Now, navigate back to the `start` directory to begin.
 
 
 // =================================================================================================
-// Enabling fault tolerance in the application
+// Enabling fault tolerance
 // =================================================================================================
 
-== Enabling fault tolerance in the application
+== Enabling fault tolerance
 
 The MicroProfile Fault Tolerance API is added as a dependency to your `pom.xml` file. Look for
-the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID. Use this feature
-to build fault-tolerant microservices with the MicroProfile Fault Tolerance API.
+the dependency with the `org.eclipse.microprofile.fault-tolerance` group ID.
+Add this dependency to use MicroProfile Fault Tolerance API in your code to use the
+fault tolerance policies in your microservices.
+
 The `mpFaultTolerance-1.0` feature is also enabled in the `src/main/liberty/config/server.xml`
-file.
+file to enables MicroProfile Fault Tolerance support in Open Liberty
```

Comment:
Enable the `mpFaultTolerance-1.0` feature is your `src/main/liberty/config/server.xml` server configuration to turn on MicroProfile Fault Tolerance in Open Liberty, which has been done for you.

Reply:
Made the change, but I think ID can help review this sentence.